### ISSUE TITLE
model.restore

### ISSUE THREAD

**Issue by Xilun1995** [lululxvi/deepxde#254]
Hi lulu,
  I am trying to implement model.restore. However, my code only works on the Mac instead of Windows  and I got this error :
![image](https://user-images.githubusercontent.com/75771241/113070730-4773f800-9191-11eb-85da-7ccd157414af.png)

Do you have any idea to make sure that it works on windows ? Since I just have gpu on my windows PC. Thanks in advance/

**Comment by Xilun1995** [comment]
One more question: 
 I change the sturcture of network as below. But I found out when I save and restore the network. It seems that it only restore [3] + [3] * 1 + [3] instead of whole network. Do you have any idea to solve this bug?
`
net = dde.maps.FNN([3] + [3] * 1 + [3], "tanh", "Glorot normal")

net.apply_output_transform(
lambda x, y : tf.concat(
[tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(x[:,0:3], 40, tf.nn.tanh), 40, tf.nn.tanh), 40, tf.nn.tanh), 30, tf.nn.tanh), 30, tf.nn.tanh), 30, tf.nn.tanh), 30, tf.nn.tanh), 1)
,tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(x[:,0:3], 40, tf.nn.tanh), 40, tf.nn.tanh), 40, tf.nn.tanh), 30, tf.nn.tanh), 30, tf.nn.tanh), 30, tf.nn.tanh), 30, tf.nn.tanh), 1)
,tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(tf.layers.dense(x[:,0:2], 30, tf.nn.tanh), 20, tf.nn.tanh), 30, tf.nn.tanh), 30, tf.nn.tanh), 1)], axis = 1
) )
`

**Comment by lululxvi** [comment]
- I don't run on Windowns, so I am not sure what happened on Windows.
- It only resotres all the weights and biases of the network. Those transforms will not be saved and restored. You need to re-define them.

**Comment by tsarikahin** [comment]
I think you defined `model` two times in the same code. So you should define the model once and the problem should be solved.
